### DISCUSSION TITLE
[월드 1.1.5] room.RemoveMessageHandler 룸 이벤트 제거

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#590] [5. Q&A]
room.RemoveMessageHandler 는 없을까요?

다음과 같이 이벤트를 등록후 
![image](https://user-images.githubusercontent.com/50188795/161892970-d1744a20-5bda-476f-87d4-6f36d1f1359d.png)

개체 인스턴스가 제거 될때 이벤트를 삭제 하고싶습니다 
![image](https://user-images.githubusercontent.com/50188795/161893068-ec36689a-521d-414e-a417-e2acfdbdf6ec.png)

어떻게 해야 할까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 질문 주신 room.RemoveMessageHandler 는 제공하고 있지 않습니다. 
추후에 지원 가능 여부에 관해 내부에서 논의해 보겠습니다. 
감사합니다.
